test cases/objc/5 objfw/meson.build

```diff
@@ -0,0 +1,13 @@
+project('TestApplication', 'objc',
+  version: '0.1',
+  default_options: ['warning_level=3'])
+
+objfw_dep = dependency('objfw', required: false)
+if not objfw_dep.found()
+  error('MESON_SKIP_TEST: Need objfw dependency')
+endif
+
+exe = executable('TestApplication', 'TestApplication.m',
+  dependencies: [objfw_dep])
+
+test('basic', exe)
```

Comment:
You don't need these test invocations

Reply:
Removed.

Reply:
Technically you might, if you're concerned that the ability to successfully compile and link the test application isn't representative of it being working and functional.

Usually we assume that linking is a good success indicator, because compilers are good at falling over horribly at even the slightest issue.

test() is then typically used for cases where we need to e.g. run a python script that compares a test fixture with a configure_file() output to assert that configure_file has any notion of how to emit the correct format.